README.md

```diff
@@ -204,28 +204,22 @@ The following are tutorials on KubeLinter written by users. If you have one that
 * [How to add a new check in KubeLinter?](https://www.psaggu.com/upstream-contribution/2021/08/17/notes.html) by Priyanka Saggu.
 * [Extending kube-linter To Build A Custom Template](https://github.com/garethahealy/kubelinter-extending-blog) by [Gareth Healy](https://github.com/garethahealy).
 
-## Community
-
-If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
-
-To contribute, check out our [contributing guide](./CONTRIBUTING.md).
```

Comment:
While this is somehow duplicated in the alpha release warning, I'd keep that line in the Community section.

Reply:
Agree, that's why there is the link to the guide in the community section below : ). Unless you insist on keeping the exact wording.

Reply:
I have no strong opinion there. The link in the rendered page is probably eye-catching enough.